Server/VikingVault.API/Controllers/TransactionController.cs

```diff
@@ -29,5 +30,18 @@ public IEnumerable<Transaction> Get()
             string userId = tokenObject.Payload["Id"].ToString();
             return _transactionService.GetTransactions(userId);
         }
+
+        [HttpPost]
+        public ActionResult Post([FromBody] Transaction transaction)
+        {
+            try
+            {
+                return Ok(_transactionService.AddTransaction(transaction));
+            }
+            catch (TransactionException e)
+            {
+                return StatusCode(500, "Internal server error");
```

Comment:
If an uncaught exception occurs during a controller, a 500 Internal server error will be returned by default, so this is not needed UNLESS you wrote it here to be explicit

Reply:
We decided together on that error code, so it's explicit. 

404 is used when an user was not found, for example, and 500 for other errors that could appear. And 200 for successful requests.